modules/kube-controller-sg/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * ## Kube Controller Security Group
+ *
+ * Define a security group for kube controller
+ *
+ */
+
+variable "cidr_blocks" {
+  description = "List of CIDR block ranges that the SG allows ingress from"
+  type        = "list"
+}
+
+variable "name" {
+  description = "Name of the Kube Controller security group"
+}
+
+module "kube-controller-sg" {
+  source      = "../security-group-base"
+  name        = "${var.name}"
+  vpc_id      = "${module.vpc.vpc_id}"
```

Comment:
There isn't a `vpc` module in this security group module. This should be `var.vpc_id`.